tcl/db_introspection.tcl

```diff
@@ -439,4 +439,18 @@ proc qc::db_extension_exists {extension_name} {
 	return true
     }
 
+}
+
+proc qc::db_is_superuser {} {
```

Comment:
Proc changed as suggested

On 9 August 2016 at 11:59, Bernhard van Woerden <notifications@github.com>
wrote:

> In tcl/db_introspection.tcl
> <https://github.com/qcode-software/qcode-tcl/pull/410#discussion_r74037294>
> :
>
> > @@ -439,4 +439,18 @@ proc qc::db_extension_exists {extension_name} {
> >  	return true
> >      }
> >
> > +}
> > +
> > +proc qc::db_is_superuser {} {
>
> I'd prefer
>
> qc::db_user_is_superuser {username} {
>
> }
>
> and get caller to fin out current user.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qcode-software/qcode-tcl/pull/410/files/e6c1ef624faa1a67aff64df61b35d58b1d9075c5#r74037294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS-1f6YsexQmpk6n0vhLlqZ5kjhIpDscks5qeF2WgaJpZM4Jf5yM>
> .
>